packages/compass/webpack.config.js

```diff
@@ -57,6 +57,12 @@ module.exports = (_env, args) => {
     },
   };
 
+  const snapshot = {
+    unmanagedPaths: [
+      path.resolve('..', '..', 'node_modules', '@mongosh', 'browser-repl'),
```

Comment:
This is useful in combination with `packages/browser-repl/scripts/sync-to-compass.js` added to mongosh [here](https://github.com/mongodb-js/mongosh/pull/2284).